frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts

```diff
@@ -61,12 +61,11 @@ export const useDashboardUrlParams = ({
   });
 
   useEffect(() => {
-    const hashOptions = parseHashOptions(
-      location.hash,
-    ) as DashboardUrlHashOptions;
-
-    setFont(hashOptions.font ?? font);
-  }, [font, location.hash, setFont]);
+    const { font } = parseHashOptions(location.hash) as DashboardUrlHashOptions;
+    if (font) {
+      setFont(font);
```

Comment:
There's no point in calling `setFont` using the `font` from `useEmbedFont` since we already cover that in an effect inside that hook
https://github.com/metabase/metabase/blob/19cbeacb939ed9d6953b9de804f89ff041b5bdf1/frontend/src/metabase/dashboard/hooks/use-embed-font.ts#L23-L25

We only want to set a new font _if_ there's a new font to set, in this case, from the URL parameter.

Also, the `setFont` dispatches a Redux action, so I'm reluctant to call it unnecessarily. If this is a setter of a state, setting the same value won't trigger the rerender so I could leave it as is. Doing this ensures we don't have to use the `font` from `useEmbedFont` in this hook result in 1 less dependency for the hook, and simplifies it.

https://github.com/metabase/metabase/blob/19cbeacb939ed9d6953b9de804f89ff041b5bdf1/frontend/src/metabase/dashboard/hooks/use-embed-font.ts#L12-L21